distribution/changelog.txt

```diff
@@ -49,6 +49,7 @@
 - Fix: [#22152] [Plugin] Negative signed integers are truncated.
 - Fix: [#22161] Using arrow keys in textboxes crashes the game.
 - Fix: [#22174] Cheats are reset when starting a server.
+- Fix: [#22185] Set nausea and intensity after test results are invaildated.
```

Comment:
_How_ a bug has been fixed is irrelevant to the user. The changelog should list the problem that has been fixed:

```suggestion
- Fix: [#22185] Intensity and nausea are incorrectly sorted in the rides list after ratings invalidation.
```